postgres/helpers/init.sql

```diff
@@ -1,3 +1,4 @@
+CREATE EXTENSION IF NOT EXISTS ltree;
```

Comment:
The extension is already enabled in `create-db.sh` script, it looks redundant.

Reply:
The reason for that is described in 9d1a399645a361b4a4848dbae82b3d4b74fcf27f 

> fix: Allow tests to work in the local and CI env

> They require the ltree extension which can only be installed as a superuser. We are thus forced to do so when the rsrdb is first created.